### PR TITLE
Added Tenant subscription

### PR DESCRIPTION
**Pull request by cheina97**
Components modified

- DashboardLogic

This PR add the subscription to the tenant query contained in DashboardLogic

### CODE REVIEW

File: frontend/src/components/workspaces/DashboardLogic/DashboardLogic.tsx [hunk 1]
```diff
@@ -1,25 +1,41 @@
...
-  //startPolling(20000);
+  useEffect(() => {
+    if (!loading) {
+      subscribeToMore({
+        variables: { tenantId: userId ?? '' },
+        document: updatedTenant,
+        updateQuery: (prev, { subscriptionData }) => {
+          if (!subscriptionData.data) return prev;
+          setData(subscriptionData.data);
+          return subscriptionData.data;
```
Comment (QcFe) [inline frontend/src/components/workspaces/DashboardLogic/DashboardLogic.tsx:27]: I would also extract `data` (can be done in the lambda definition directly, something like `{ { data } : subscriptionData }`, I'm not really sure about the syntax order though)

**Review by QcFe — APPROVED** [review]
looks alright, just a minor comment

File: frontend/src/components/workspaces/DashboardLogic/DashboardLogic.tsx (hunk 1 above)
Comment (cheina97) [reply frontend/src/components/workspaces/DashboardLogic/DashboardLogic.tsx:27]: I did it

**Pull request merged**